### PR TITLE
Update to latest version of ah_var_store gatk override jar

### PR DESCRIPTION
**Pull request by gbggrant**

### CODE REVIEW

File: scripts/variantstore/wdl/GvsCreateFilterSet.wdl [hunk 1]
```diff
@@ -14,7 +14,7 @@ workflow GvsCreateFilterSet {
     Array[String] snp_recalibration_annotation_values = ["AS_QD", "AS_MQRankSum", "AS_ReadPosRankSum", "AS_FS", "AS_MQ", "AS_SOR"]
 
     File interval_list = "gs://gcp-public-data--broad-references/hg38/v0/wgs_calling_regions.hg38.noCentromeres.noTelomeres.interval_list"
-    File gatk_override = "gs://broad-dsp-spec-ops/scratch/bigquery-jointcalling/jars/ah_var_store_20220406/gatk-package-4.2.0.0-480-gb62026a-SNAPSHOT-local.jar"
+    File gatk_override = "gs://broad-dsp-spec-ops/scratch/bigquery-jointcalling/jars/ah_var_store_20220415/gatk-package-4.2.0.0-492-g1387d47-SNAPSHOT-local.jar"
```
Comment (mcovarr) [inline scripts/variantstore/wdl/GvsCreateFilterSet.wdl:17]: this is at least a couple of revisions back now so it wouldn't be picking up the zero-padding improvements from #7783

Comment (gbggrant) [reply scripts/variantstore/wdl/GvsCreateFilterSet.wdl:17]: True - I'll rebuild it - but the only gatk jar improvements in 7783 were really just cleaning up some logging.

Comment (mcovarr) [reply scripts/variantstore/wdl/GvsCreateFilterSet.wdl:17]: oh yeah you're right... wow did I misremember that code 😅

**Review by mcovarr — APPROVED** [review]

File: scripts/variantstore/wdl/GvsCreateFilterSet.wdl (hunk 1 above)
Comment (RoriCremer) [reply scripts/variantstore/wdl/GvsCreateFilterSet.wdl:17]: in an ideal world, I think we would also create these ahead of our releases and have one named for the release---so we'd have a Charlie jar (though we've re-made it a few times over the last few weeks....)

**Review by RoriCremer — APPROVED** [review]

**Review by rsasch — APPROVED** [review]
thanks for this 👍🏻

**Pull request merged**